api/src/paths/project/{projectId}/attachments/upload.ts

```diff
@@ -92,7 +92,7 @@ POST.apiDoc = {
  */
 export function uploadMedia(): RequestHandler {
   return async (req, res) => {
-    defaultLog.debug({ label: 'uploadMedia', message: 'files.length', files: req?.files?.length });
+    defaultLog.debug({ label: 'uploadMedia', message: 'files', files: req.files });
```

Comment:
We cant print req.files, because that will print all of the binary information, which is potentially massive

Reply:
If we wanted to print the files, we'd need to do something like: `files: {...req.files, binary: null}`

Reply:
cool